src/components/m-datepicker/_preview.html

```diff
@@ -1,4 +1,6 @@
-<!--axa-datepicker button-ok="ok" button-cancel="abbrechen" classes="m-datepicker"></axa-datepicker-->
-<axa-container>
-  <h1>Coming soon...</h1>
-</axa-container>
+<axa-m-datepicker button-ok="ok" button-cancel="abbrechen" locale="de-ch" classes="m-datepicker"></axa-m-datepicker>
+<axa-m-datepicker output-iso button-ok="Ja" button-cancel="abbrechen" classes="m-datepicker"></axa-m-datepicker>
```

Comment:
how about validations like, min-date, max-date?
Or will this be part of the organism, still I guess somehow it needs to be incorporated here 🤔

Reply:
Validation works here that we cannot choose more dates than allowed in the dropdown (so no validation cause u cannot choose what is not allowed).

The input text instead will have validation for user manual input

Reply:
example: `<axa-m-datepicker lower-end-year="1980" higher-end-year="2018" start-month="11" locale="it-ch" button-ok="ok" button-cancel="abbrechen" classes="m-datepicker"></axa-m-datepicker>`

Reply:
the calendar wont allow u to select 2019 for example

Reply:
Okay, I see.
It would be nice though, to describe a full date, including days for min/max or start/end

Reply:
we will double check the start-end date. that would mean that on the calendar some current month days will be grey as well and therefore need a design approval